assistant/run.prod.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
 set -eu
 
+python -m seeder.generate_prompt_sqlite
```

Comment:
This resets to the seeder prompts every time the system runs this script. The database should be external to the container and seeding should be done either when the database is found to be empty or left to an admin on installation.

Reply:
I misunderstood. My bad.